### PR TITLE
Improve interactive debugger

### PR DESCRIPTION
**Pull request by alexcb**
* start shell prompt on new line
* write last failed command to bash history
* include ash shell as potential shell

### CODE REVIEW

File: cmd/debugger/main.go [hunk 1]
```diff
@@ -91,6 +101,8 @@ func interactiveMode(ctx context.Context, remoteConsoleAddr string) error {
 		return err
 	}
 
+	_ = populateShellHistory(cmd) // best effort
```
Comment (vladaionescu) [inline cmd/debugger/main.go:104]: Good idea

File: go.mod [hunk 2]
```diff
@@ -3,89 +3,37 @@ module github.com/earthly/earthly
...
 	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
-	google.golang.org/api v0.0.0-20160322025152-9bf6e6e569ff // indirect
-	google.golang.org/cloud v0.0.0-20151119220103-975617b05ea8 // indirect
-	gopkg.in/alecthomas/kingpin.v2 v2.2.6 // indirect
 	gopkg.in/yaml.v2 v2.2.8
-	k8s.io/kubernetes v1.13.0 // indirect
 )
 
 replace (
+	github.com/alessio/shellescape => github.com/alexcb/shellescape v0.0.0-20200921195046-bf72418e9bfb
```
Comment (vladaionescu) [inline go.mod:32]: What's the reason for the fork?

**Review by vladaionescu — APPROVED** [review]
:+1:

File: go.mod (hunk 2 above)
Comment (alexcb) [reply go.mod:32]: waiting for my PR to get merged: https://github.com/alessio/shellescape/pull/6

**Pull request merged**